components/authz-service/storage/postgres/migration/sql/45_add_application_policies.up.sql

```diff
@@ -0,0 +1,24 @@
+BEGIN;
+
+INSERT INTO policies
+    VALUES ('aee14d59-da0b-4974-ba6d-1a018b024874',
+            '{"action": "*", "effect": "allow", "resource": "service_groups", "subjects": ["user:*"]}',
+            CURRENT_TIMESTAMP,
+            1,
+            TRUE)
+    ON CONFLICT (id) DO UPDATE
+    SET policy_data='{"action": "list", "effect": "allow", "resource": "service_groups", "subjects": ["user:*"]}',
+        deletable=TRUE;
+
+UPDATE iam_roles
```

Comment:
ah i forgot to mention one thing 😅 
hopefully last thing to fix: move L13-23 into a new migration file under the `postgres/datamigrations/sql` [directory](https://github.com/chef/automate/tree/master/components/authz-service/storage/postgres/datamigration/sql)